pkg/ccl/importccl/import_stmt.go

```diff
@@ -251,6 +247,22 @@ func importPlanHook(
 		if err != nil {
 			return err
 		}
+
+		// Certain ExternalStorage URIs require super-user access. Check all the
+		// URIs passed to the IMPORT command.
+		for _, file := range filenamePatterns {
+			requiresAdmin, err := cloudimpl.URIRequiresAdminRole(file)
```

Comment:
I'd have it return the scheme or something we can sprintf into the admin check below, so it would read `IMPORT from http URI requires admin" instead of just IMPORT since that isn't quite true.